lpcompiler.pas

```diff
@@ -1380,6 +1387,9 @@ function TLapeCompiler.ParseMethodHeader(out Name: lpString; addToScope: Boolean
 
           if (Tokenizer.Tok = tk_sym_Equals) then
           begin
+            if (Result.MethodDef = mdProperty) then
+              LapeException(lpeDefaultParamInProperties, Tokenizer.DocPos);
```

Comment:
That would get really messy when having several different array indexing properties.
> a.index[0];
> a.Index[0,0];
.. etc.